docs/user-guide/cli-configuringcli-evfile.md

```diff
@@ -2,15 +2,22 @@
 
 If it is not possible to configure your own system environment variables, create a special configuration file to set these variables for Zowe CLI commands.
 
+:::info Required roles: Security administrator, DevOps architect
+:::
+
 Although not common, there are cases where users do not have the ability to configure their own system environment variables. This can happen when users are working on hosted integrated development environments (IDEs), or in a highly locked down environment.
 
 When working under these kinds of restrictions, you can set environment variables that apply to CLI commands. To do this, create a `.zowe.env.json` file storing key-value pairs that specify your configurations.
 
-**Note:** Use a `.zowe.env.json` file *only* when it is not possible to set your own system environment variables. If you are able to configure environment variables in your system, continue to do so.
+:::note
+
+Use a `.zowe.env.json` file *only* when it is not possible to set your own system environment variables. If you are able to configure environment variables in your system, continue to do so.
+
+:::
 
 ## How `.zowe.env.json` works
 
-When a Zowe command is issued, the command initializes the Imperative CLI Framework so that it loads all the utilities that allow the command to function. Imperative reads the `.zowe.env.json` configuration file and sets the environment variables before any loggers or Zowe CLI finish their own initialization.
+When a Zowe command is issued, the command initializes the Imperative CLI Framework **[is Imperative something still used in V3?]**so that it loads all the utilities that allow the command to function. Imperative reads the `.zowe.env.json` configuration file and sets the environment variables before any loggers or Zowe CLI finish their own initialization.
```

Comment:
@adam-wolfe @zFernand0 - should we be mentioned Imperative here for V3? If not, how to explain this w/o Imperative?